runtime/moonbeam/src/lib.rs

```diff
@@ -723,6 +723,7 @@ impl pallet_author_mapping::Config for Runtime {
 	type Event = Event;
 	type DepositCurrency = Balances;
 	type DepositAmount = ConstU128<{ 100 * currency::GLMR * currency::SUPPLY_FACTOR }>;
+	type Keys = NimbusId; // TODO: consider making custom type?
```

Comment:
as I mentioned before, I think we need to try to envision what these type is gonna be in the future. Right now its a single NimbusId, which I am assuming links to a single additional key. Is this supposed to be the case in the future?

Reply:
Sorry just saw your previous answer. If it is a tuple, we need to take into account for the precompiles too